base_layer/wallet_ffi/src/lib.rs

```diff
@@ -2836,6 +2837,114 @@ pub unsafe extern "C" fn pending_inbound_transaction_destroy(transaction: *mut T
 
 /// -------------------------------------------------------------------------------------------- ///
 
+/// ----------------------------------- Transport Send Status -----------------------------------///
+
+/// Gets the direct send status of a TariTransactionSendStatus
+///
+/// ## Arguments
+/// `status` - The pointer to a TariTransactionSendStatus
+/// `error_out` - Pointer to an int which will be modified to an error code should one occur, may not be null. Functions
+/// as an out parameter.
+///
+/// ## Returns
+/// `c_uint` - Returns 1 if the transaction was sent directly, zero if it was not
```

Comment:
This should be a bool

Reply:
but actually I would say that to save having to create and maintain so many extra FFI methods why not just a single method that returns the enum encoded as unit?